src/bin/import_cards.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+import json
+from pathlib import Path
+import typing
+
+import fixpath as _
+from config import get_settings
+from es_client import get_es_client
+
+"""
+Recursively import all .json files from the cards directory
+"""
+
+settings = get_settings()
+stats = {
+    'created': 0,
+    'updated': 0,
+    'failed': 0
+}
+
+
+def _import_card(card_path: str, client) -> None:
+    """
+    Import a single .json card
+    """
+    with open(card_path, "r", encoding='utf-8') as json_file:
+        card_json = json.load(json_file)
+        es_json = _transform_card_json(card_json)
+        resp = client.index(index="cards", body=es_json, id=es_json['id'])
+
+        match resp['result']:
+            case 'created':
+                stats['created'] += 1
+            case 'updated':
+                stats['updated'] += 1
+            case _:
+                stats['failed'] += 1
+                # TODO Log errors?
+                print(card_path + " failed")
```

Comment:
Je verrais plus quelque chose comme ça
```suggestion
       if resp['result'] in ["created", "updated"]:
           stats[resp["result']] += 1
       else:
           stats['failed'] += 1
           # TODO Log errors?
           print(card_path + " failed")
```

Le switch case est très rarement utilisé en Python, on se rend compte qu'un dict fait la même chose.

Reply:
Ok, je changerai
J'étais content qu'ils ajoutent enfin un switch en Python 😅

Reply:
https://codedamn.com/news/python/switch-case-statement

Quand tu viens d'un autre language c'est perturbant, mais les dict sont tellement puissant!